bindings_ffi/src/mls.rs

```diff
@@ -2181,7 +2181,10 @@ impl FfiConversation {
 #[uniffi::export]
 impl FfiConversation {
     pub fn id(&self) -> Vec<u8> {
-        self.inner.group_id.clone()
+        match self.inner.client.stitched_group(&self.inner.group_id) {
+            Ok(group) => group.group_id.clone(),
+            Err(_) => self.inner.group_id.clone(),
+        }
```

Comment:
This fixes a lot of weirdness with dm stitching letting dms now always return the correct id after they've been synced.

Reply:
Wanted to confirm that there was no real performance impacts for this change
Previously it .000000s
now its 0.000029s